README.md

```diff
@@ -48,6 +48,8 @@ yarn serve:app
 
 ```bash
 yarn build:app
+cd dist
+npx lite-server
```

Comment:
Please change to, the production build is generated in the `dist` directory which can be served using caddy or nginx and don't mention lite-server at all.

Reply:
should be resolved in : [4989227](https://github.com/threefoldtech/grid_weblets/pull/1216/commits/4989227ae33783945e0639aba19d67741d25bbc6)